lib/core/src/lib/form/components/form-renderer.component.html

```diff
@@ -16,56 +19,43 @@
 
 <ng-template #render let-fieldToRender="fieldToRender">
     <div *ngFor="let currentRootElement of fieldToRender">
-        <div
-            *ngIf="currentRootElement.type === 'container' || currentRootElement.type === 'group'"
-            class="adf-container-widget"
-            [id]="'field-' + currentRootElement?.id + '-container'"
-            [hidden]="!currentRootElement?.isVisible"
-        >
+        <div *ngIf="currentRootElement.type === 'container' || currentRootElement.type === 'group'"
+             class="adf-container-widget"
+             [id]="'field-'+currentRootElement?.id+'-container'"
+             [hidden]="!currentRootElement?.isVisible">
             <div [hidden]="!currentRootElement?.isGroup()" class="adf-container-widget__header">
-                <h4 class="adf-container-widget__header-text" id="container-header" [class.adf-collapsible]="currentRootElement?.isCollapsible()">
-                    <button
-                        *ngIf="currentRootElement?.isCollapsible()"
-                        mat-icon-button
-                        class="mdl-button--icon"
-                        (click)="onExpanderClicked(currentRootElement)"
-                    >
+                <h4 class="adf-container-widget__header-text"
+                    id="container-header"
+                    [class.adf-collapsible]="currentRootElement?.isCollapsible()">
+                    <button *ngIf="currentRootElement?.isCollapsible()"
+                            mat-icon-button
+                            class="mdl-button--icon"
+                            (click)="onExpanderClicked(currentRootElement)">
                         <mat-icon>{{ currentRootElement?.isExpanded ? 'expand_more' : 'expand_less' }}</mat-icon>
                     </button>
-                    <span
-                        (click)="onExpanderClicked(currentRootElement)"
-                        role="button"
-                        tabindex="0"
-                        (keyup.enter)="onExpanderClicked(currentRootElement)"
-                        [id]="'container-header-label-' + currentRootElement?.id"
-                        >{{ currentRootElement.name | translate }}</span
-                    >
+                    <span (click)="onExpanderClicked(currentRootElement)"
+                          [id]="'container-header-label-'+currentRootElement?.id">
```

Comment:
There are a couple of missing properties here (role, tabindex, keyup.enter). Are those intentional

Reply:
I don't know, I was just formatting this line